tests/reference/tests/chocolateyfeature/fail_to_remove_feature.rb

```diff
@@ -0,0 +1,25 @@
+require 'chocolatey_helper'
+test_name 'MODULES-3034 - Attempt to remove feature'
+confine(:to, :platform => 'windows')
+
+backup_config
+
+# arrange
+chocolatey_src = <<-PP
+  chocolateyfeature {'checksumFiles':
+    ensure => absent,
+  }
+PP
+
+# teardown
+teardown do
+  reset_config
+end
+
+# act
+step 'Apply manifest'
+apply_manifest(chocolatey_src, :expect_failures => true) do
+  step 'Verify Failure'
+  assert_match(/Error: Validation of Chocolateyfeature\[checksumFiles\] failed: asdf/, stderr, "stderr did not match expected")
```

Comment:
WIP - this is on purpose to cause an error - Want to find out what the value is to add but was not able to